### PR TITLE
[SID-1095] Sourceful beta

### PR DESCRIPTION
**Pull request by ikovic**
## Description

[YouTrack issue](https://todo.irdcorp.dev/issue/SID-000)

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have generated the new version of the docs website and smoke tested it
- [ ] I have checked that my changes haven't caused semantic errors in the existing docs
- [ ] I have updated the README and DEVELOPMENT files

### CODE REVIEW

File: packages/react/src/components/form/initial.tsx [hunk 1]
```diff
@@ -87,7 +87,11 @@ const Oidc: React.FC<OidcProps> = ({ providers, handleClick }) => {
             icon={PROVIDER_TO_ICON[p.options?.provider]}
           >
             {text["initial.oidc"]}
-            <span className={styles.oidcProvider}>{p.options?.provider}</span>
+            <span className={styles.oidcProvider}>
+              {p.options?.provider === "azuread"
+                ? "Microsoft Account"
+                : p.options?.provider}
+            </span>
```
Comment (paulo-raca) [inline packages/react/src/components/form/initial.tsx:94]: I think we recently added provider names in the i18n table?

File: packages/react/src/components/form/initial.tsx [hunk 2]
```diff
@@ -87,7 +87,11 @@ const Oidc: React.FC<OidcProps> = ({ providers, handleClick }) => {
             icon={PROVIDER_TO_ICON[p.options?.provider]}
           >
             {text["initial.oidc"]}
-            <span className={styles.oidcProvider}>{p.options?.provider}</span>
+            <span className={styles.oidcProvider}>
+              {p.options?.provider === "azuread"
```
Comment (ikovic) [inline packages/react/src/components/form/initial.tsx:91]: @ikovic TODO - expose a way to override the OIDC provider names for the buttons

File: packages/react/src/components/form/initial.tsx (hunk 1 above)
Comment (ikovic) [reply packages/react/src/components/form/initial.tsx:94]: Yep, in the dashboard :)

**Comment by ikovic** [comment]
Closed in favour of #117

**Pull request closed without merging**